### ISSUE TITLE
How to convert to onnx model with spatial==1?

### ISSUE THREAD

**Issue by luan1412167** [apache/incubator-mxnet#16439]
Note: Providing complete information in the most concise form is the best way to get help. This issue template serves as the checklist for essential information to most of the technical issues and bug reports. For non-technical issues and feature requests, feel free to present the information in what you believe is the best form.

For Q & A and discussion, please start a discussion thread at https://discuss.mxnet.io 

## Description
I'm using arcface model, convert it to onnx format. With spatial of Batch Norm = 0, I can't run it on ONNX runtime. 
When I set spatial=1 in file "/mxnet/contrib/onnx/mx2onnx/_op_translations.py" I can run it but with wrong result. 
Whether ONNX converter from mxnet to onnx have get onnx model with spatial=1 and get right result?

## Environment info (Required)

```
----------Python Info----------
Version      : 3.7.4
Compiler     : GCC 7.3.0
Build        : ('default', 'Aug 13 2019 20:35:49')
Arch         : ('64bit', '')
------------Pip Info-----------
Version      : 19.2.3
Directory    : /home/luandd/miniconda3/envs/luandao/lib/python3.7/site-packages/pip
----------MXNet Info-----------
Version      : 1.5.0
Directory    : /home/luandd/miniconda3/envs/luandao/lib/python3.7/site-packages/mxnet
Commit Hash   : 75a9e187d00a8b7ebc71412a02ed0e3ae489d91f
Library      : ['/home/luandd/miniconda3/envs/luandao/lib/python3.7/site-packages/mxnet/libmxnet.so']
Build features:
✖ CUDA
✖ CUDNN
✖ NCCL
✖ CUDA_RTC
✖ TENSORRT
✔ CPU_SSE
✔ CPU_SSE2
✔ CPU_SSE3
✔ CPU_SSE4_1
✔ CPU_SSE4_2
✖ CPU_SSE4A
✔ CPU_AVX
✖ CPU_AVX2
✖ OPENMP
✖ SSE
✔ F16C
✖ JEMALLOC
✖ BLAS_OPEN
✖ BLAS_ATLAS
✖ BLAS_MKL
✖ BLAS_APPLE
✔ LAPACK
✖ MKLDNN
✔ OPENCV
✖ CAFFE
✖ PROFILER
✔ DIST_KVSTORE
✖ CXX14
✖ INT64_TENSOR_SIZE
✔ SIGNAL_HANDLER
✖ DEBUG
----------System Info----------
Platform     : Linux-5.0.0-29-generic-x86_64-with-debian-buster-sid
system       : Linux
node         : vid-luandd-u
release      : 5.0.0-29-generic
version      : #31~18.04.1-Ubuntu SMP Thu Sep 12 18:29:21 UTC 2019
----------Hardware Info----------
machine      : x86_64
processor    : x86_64
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              8
On-line CPU(s) list: 0-7
Thread(s) per core:  2
Core(s) per socket:  4
Socket(s):           1
NUMA node(s):        1
Vendor ID:           GenuineIntel
CPU family:          6
Model:               142
Model name:          Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
Stepping:            12
CPU MHz:             4085.437
CPU max MHz:         4600,0000
CPU min MHz:         400,0000
BogoMIPS:            3984.00
Virtualization:      VT-x
L1d cache:           32K
L1i cache:           32K
L2 cache:            256K
L3 cache:            8192K
NUMA node0 CPU(s):   0-7
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf tsc_known_freq pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb invpcid_single ssbd ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid mpx rdseed adx smap clflushopt intel_pt xsaveopt xsavec xgetbv1 xsaves dtherm ida arat pln pts hwp hwp_notify hwp_act_window hwp_epp md_clear flush_l1d arch_capabilities
----------Network Test----------
Setting timeout: 10
Timing for MXNet: https://github.com/apache/incubator-mxnet, DNS: 0.0082 sec, LOAD: 1.3817 sec.
Timing for Gluon Tutorial(en): http://gluon.mxnet.io, DNS: 0.2195 sec, LOAD: 0.3970 sec.
Timing for Gluon Tutorial(cn): https://zh.gluon.ai, DNS: 0.3286 sec, LOAD: 0.8446 sec.
Timing for FashionMNIST: https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/dataset/fashion-mnist/train-labels-idx1-ubyte.gz, DNS: 1.1782 sec, LOAD: 0.9432 sec.
Timing for PYPI: https://pypi.python.org/pypi/pip, DNS: 0.0704 sec, LOAD: 1.1657 sec.
Timing for Conda: https://repo.continuum.io/pkgs/free/, DNS: 0.0738 sec, LOAD: 1.3407 sec.
----------Environment----------


```

Package used (Python/R/Scala/Julia):
(I'm using Python3.7)

## Build info (Required if built from source)

Compiler (gcc/clang/mingw/visual studio):

MXNet commit hash:
(Paste the output of `git rev-parse HEAD` here.)

Build config:
(Paste the content of config.mk, or the build command.)

## Error Message:
(Paste the complete error message, including stack trace.)

## Minimum reproducible example
(If you are using your own code, please provide a short script that reproduces the error. Otherwise, please provide link to the existing example.)

## Steps to reproduce
(Paste the commands you ran that produced the error.)

1.
2.

## What have you tried to solve it?

1. onnx runtime not support spatial Batch Norm = 0. Change spatial=1 in file "/mxnet/contrib/onnx/mx2onnx/_op_translations.py". I can run the model. However the result is wrong.
2.

**Comment by haojin2** [comment]
@anirudhacharya Any comments?

**Comment by anirudhacharya** [comment]
while converting onnx models, the`batchnorm` operator has the spatial parameter disabled due to the way `batchnorm` is defined in onnx - https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/contrib/onnx/mx2onnx/_op_translations.py#L357-L359

changing the label to "feature request"

**Comment by TristonC** [comment]
@Zha0q1  Has this issue been solved?

**Comment by Zha0q1** [comment]
This use case should be supported in the upcoming mxnet 1.9 release
